packages/console/src/pages/Dashboard/index.tsx

```diff
@@ -23,35 +33,102 @@ const Dashboard = () => {
+              <div className={styles.title}>{t('dashboard.total_users')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{totalData.totalUserCount}</div>
               </div>
-            </div>
-          </Card>
-          <Card className={styles.block}>
-            <div className={styles.title}>{t('dashboard.new_users_7_days')}</div>
-            <div className={styles.content}>
-              <div className={styles.number}>{newData.last7Days.count}</div>
-              <div className={classNames(styles.delta, newData.last7Days.delta < 0 && styles.down)}>
-                ({newData.last7Days.delta > 0 && '+'}
-                {newData.last7Days.delta})
-                {newData.last7Days.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+            </Card>
+            <Card className={styles.block}>
+              <div className={styles.title}>{t('dashboard.new_users_today')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{newData.today.count}</div>
+                <div className={classNames(styles.delta, newData.today.delta < 0 && styles.down)}>
+                  ({newData.today.delta > 0 && '+'}
+                  {newData.today.delta}){newData.today.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                </div>
               </div>
+            </Card>
+            <Card className={styles.block}>
+              <div className={styles.title}>{t('dashboard.new_users_7_days')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{newData.last7Days.count}</div>
+                <div
+                  className={classNames(styles.delta, newData.last7Days.delta < 0 && styles.down)}
+                >
+                  ({newData.last7Days.delta > 0 && '+'}
+                  {newData.last7Days.delta})
+                  {newData.last7Days.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                </div>
+              </div>
+            </Card>
+          </div>
+          <Card>
+            <div className={styles.dau}>
+              <div className={styles.title}>{t('dashboard.daily_active_users')}</div>
+              <div className={styles.content}>
+                <div className={styles.number}>{activeData.dau.count}</div>
+                <div className={classNames(styles.delta, activeData.dau.delta < 0 && styles.down)}>
+                  ({activeData.dau.delta > 0 && '+'}
+                  {activeData.dau.delta}){activeData.dau.delta > 0 ? <ArrowUp /> : <ArrowDown />}
+                </div>
+              </div>
+            </div>
+            <div className={styles.curve}>
+              <ResponsiveContainer>
+                <AreaChart
+                  data={activeData.dauCurve.map((item) => ({
+                    ...item,
+                    // Remove "year" for a compact label.
+                    date: item.date.replace(/\d{4}-/, ''),
```

Comment:
nit: Remenber we have `day.js` in our console.
```js
dayjs(item.date).format('MM-DD');
```

Reply:
The `date` returned from API does not contain timezone info, so is not safe to parse it into a date.